src/components/invest/index.js

```diff
@@ -7,17 +7,23 @@ import { noMetaMaskAlert, noContractAlert, investmentDisabledAlert, investmentDi
 import { Loader } from '../Common/Loader'
 import { ICOConfig } from '../Common/config'
 import { defaultState, GAS_PRICE } from '../../utils/constants'
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import QRCode from 'qrcode.react';
 
 export class Invest extends React.Component {
   constructor(props) {
       super(props);
+      this.pristineTokenInput = true;
       window.scrollTo(0, 0);
       if (this.tokensToInvestOnChange.bind) this.tokensToInvestOnChange = this.tokensToInvestOnChange.bind(this);
       if (this.investToTokens.bind) this.investToTokens = this.investToTokens.bind(this);
       var state = defaultState;
       state.seconds = 0;
       state.loading = true;
       state.pristineTokenInput = true;
+      state.investThrough = 'metamask'
+      state.web3Available = false;
+      state.crowdsaleAddress = ICOConfig.crowdsaleContractURL || getURLParam("addr")
```

Comment:
Extract lines 21-26 to an object in a constant or util file and use Object.assign to add it to the default state on line 20